packages/ui/src/memberships/components/CurrentMember/CurrentMember.tsx

```diff
@@ -17,7 +18,7 @@ export const CurrentMember = () => {
   if (!hasMembers) {
     return (
       <MembershipButtonsWrapper>
-        <AddMembershipButton size="large">Create membership</AddMembershipButton>
+        <AddMembershipButton size="large">Join Now</AddMembershipButton>
```

Comment:
Oh, I see, this is the reason for a global context.

Reply:
Also, me and @WRadoslaw decided to make provider global, because of `useLocalStorage` hook

When we used just `useOnBoarding` hook in several places changes in localStorage was not watched and onboarding status was never updated till page refresh